<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAIN_Sensor_Selection_Window</class>
 <widget class="QMainWindow" name="MAIN_Sensor_Selection_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sensor Selection Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
 background-color: rgb(244, 221, 192);
color:rgb(0, 0, 0);
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="sensor_select_MAX_Label">
      <property name="text">
       <string>Select x Sensors</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="Module1_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="Module2_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="Module3_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_10">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_11">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_12">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 12</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="Module4_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_13">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 13</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_14">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 14</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_15">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 15</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_16">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 16</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="Module5_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_17">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 17</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_18">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 18</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_19">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 19</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_20">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 20</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="Module6_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 6&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_21">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 21</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_22">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 22</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_23">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 23</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_24">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 24</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="Module7_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 7&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_25">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 25</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_26">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 26</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_27">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 27</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_28">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 28</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="Module8_Title">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Module 8&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_29">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 29</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_30">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 30</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_31">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 31</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Sensor_32">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sensor 32</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sensor_selection_DONE_Button">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(204, 51, 0);
color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
